src/lib/cert/cvc/ecdsa_sig.cpp

```diff
@@ -50,7 +50,7 @@ ECDSA_Signature decode_concatenation(const std::vector<byte>& concat)
 
    const size_t rs_len = concat.size() / 2;
 
-   BigInt r = BigInt::decode(&concat[0], rs_len);
+   BigInt r = BigInt::decode(concat.data(), rs_len);
```

Comment:
You are right. I tried to revert all places where the automatic change script destroyed structure but did not catch this one. I'll revert it.